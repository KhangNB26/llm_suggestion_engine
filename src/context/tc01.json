{
  "workItems": [
    {
      "work_id": "11111111-aaaa-bbbb-cccc-000000000001",
      "task_id": "11111111-aaaa-bbbb-cccc-000000000001",
      "title": "On tap Giai tich",
      "effective_deadline": "2025-09-26T15:00:00Z",
      "effective_priority": 1,
      "status": 0,
      "is_atomic": true,
      "task_deadline": "2025-09-26T15:00:00Z",
      "task_priority": 1,
      "task_status": 0
    }
  ],
  "scheduleEntries": [
    {
      "id": "22222222-aaaa-bbbb-cccc-000000000001",
      "user_id": "11111111-1111-1111-1111-111111111111",
      "work_item_id": "11111111-aaaa-bbbb-cccc-000000000001",
      "start_at": "2025-09-25T07:00:00Z",
      "planned_minutes": 90,
      "status": 0,
      "end_at": "2025-09-25T08:30:00Z"
    }
  ],
  "freeSlots": [
    {
      "startUtc": "2025-09-25T09:00:00Z",
      "endUtc": "2025-09-25T11:00:00Z",
      "startLocal": "2025-09-25T16:00:00+07:00",
      "endLocal": "2025-09-25T18:00:00+07:00",
      "durationMinutes": 120,
      "timeZone": "Asia/Ho_Chi_Minh"
    }
  ],
  "studyProfile": {
    "user_id": "11111111-1111-1111-1111-111111111111",
    "chronotype": "early_bird",
    "focusStyle": "deep_work",
    "workStyle": "solo"
  }
}
