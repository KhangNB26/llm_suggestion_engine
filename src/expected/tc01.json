{
  "items": [
    {
      "item_type": 0,
      "title": "On tap Giai tich",
      "parentTaskId": null,
      "estimated_minutes": 90,
      "deadline": "2025-09-25T11:00:00Z",
      "confidence": 0.85,
      "reason": "Chon slot buoi sang theo ho so early_bird."
    }
  ]
}
